source/views/sis/course-search/lib/format-day.js

```diff
@@ -20,3 +20,24 @@ export function formatDay(day: string) {
 			return 'Unknown Day'
 	}
 }
+
+export function formatDayAbbrev(day: string) {
+	switch (day) {
+		case 'Mo':
+			return 'M'
+		case 'Tu':
+			return 'T'
+		case 'We':
+			return 'W'
+		case 'Th':
+			return 'Th'
+		case 'Fr':
+			return 'F'
+		case 'Sa':
```

Comment:
What if we just let Sa and Su fall through to the default case, and just returned the input for `default:`?

Reply:
I added this in response to a comment by @drewvolz, are there no classes on Saturday or Sunday?

Reply:
I'll make up a list of all the timestrings in my course database to check

Reply:
But, even if there are, we'd still be returning `Sa` and `Su` for those two days, which seems reasonable to me.

Reply:
(I'm voting to get rid of "Unknown Day" entirely; if we fall through to `default:`, just return the input)

Reply:
Ohhh I got you. I like that

Reply:
Here's my list of all the course times I know of: https://gist.github.com/hawkrives/df4c1c2746594d705353acdae6530085

It contains 0 occurrences of the letter "S"

Reply:
So should I get rid of the 'Sa' and 'Su' options or nah? I will change the `default:`

Reply:
Let's go ahead and remove the Saturday and Sunday ones.